### PR TITLE
Commiting Normalization Blog to main branch #3024

### PR DESCRIPTION
**Pull request by Lucas-Jeanniot**
### Description
Completed blog on Rank Normalization in Hybrid Search. 
 
### Issues Resolved

Closes https://github.com/opensearch-project/project-website/issues/3024

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by pajuric** [comment]
@Lucas-Jeanniot - Thank you. I will get this through the final review. We will likely publish next week

**Comment by pajuric** [comment]
@Lucas-Jeanniot - Please add your bio and a headshot to the PR. Here's a link to a guide to help you configure it properly: https://github.com/opensearch-project/project-website/blob/main/BLOG_GUIDE.md

**Comment by Lucas-Jeanniot** [comment]
Hi Patti,

I noticed I already have a profile from my previous speaker roles, I have
updated the profiles appropriately (I hope)

Let me know if everything is in order.

Thanks,
Lucas

**Comment by Lucas-Jeanniot** [comment]
Hi Patti,

Wanted to follow up and see if everything was okay with the blog, I see
it's still waiting on merge approval.

Thanks,
Lucas

**Comment by pajuric** [comment]
@Lucas-Jeanniot - I have it slated for final review and scheduled for Monday.

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:359]:
```suggestion
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:364]:
```suggestion
- **Handling outliers:** L2 normalization can mitigate the effect of outliers because extreme values have less impact on the final normalized scores as compared to min-max normalization.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:371]:
```suggestion
**Potential drawbacks and when to use each method**
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:372]: Above heading: H3?

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:377]:
```suggestion
**Min-max normalization drawbacks**
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:382]:
```suggestion
- **L2 normalization:** This method is preferred when the goal is to maintain the relative importance of each score within the context of the entire dataset and when the presence of outliers needs to be minimized.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:387]:
```suggestion
**Scenarios in which L2 normalization is more effective**
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:388]:
```suggestion
- **Document ranking in search engines:** When ranking documents based on multiple relevance factors, L2 normalization ensures that no single factor disproportionately influences the final ranking.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:389]:
```suggestion
- **Recommender systems:** In scenarios where recommendations are based on multiple user preferences, L2 normalization helps balance the different preferences without allowing any single preference to dominate.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:392]:
```suggestion
**Scenarios in which min-max normalization is preferred**
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:393]:
```suggestion
- **Image processing:** In image processing tasks, min-max normalization is commonly used to scale pixel values to a standard range (for example, 0 to 1) for consistency and ease of interpretation.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:394]:
```suggestion
- **Financial data analysis:** When analyzing financial data, such as stock prices or returns, min-max normalization can be used to compare different datasets on a common scale.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:403]:
```suggestion
**Configuration and usage in an OpenSearch environment**
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:405]:
```suggestion
1. **Define the search pipeline:** Create a search pipeline with a normalization processor and specify the normalization technique:
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:433]:
```suggestion
2. **Use the search pipeline:** Use the search pipeline in search requests to apply normalization and combination techniques to search results:
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:434]:
```suggestion
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:469]:
```suggestion
**Practical tips for developers to optimize search results using normalization techniques**
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:471]:
```suggestion
1. **Choose the right technique:** Assess your data and query nature to select the most suitable normalization technique, such as L2 for varying magnitudes or min-max for fixed-range scaling.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:475]:
```suggestion
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:477]:
```suggestion
3. **Monitor performance:** Track the performance impact of normalization, especially in high-traffic environments. Optimize configurations to balance accuracy and performance.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:478]:
```suggestion
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:482]:
```suggestion
- **Identical scores:** Implement logic to handle cases in which all scores are identical to avoid division by zero or uniform scores.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:483]:
```suggestion
- **Zero scores:** Introduce a small epsilon value to avoid issues with zero scores in calculations.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:484]:
```suggestion
- **Sparse data:** Address imbalanced data distributions to prevent skewed normalization results.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:485]:
```suggestion
- **Outliers:** Apply techniques to minimize the impact of extreme values on the normalized scores.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:486]:
```suggestion
- **Empty sub-queries:** Ensure empty sub-queries are handled gracefully without affecting the overall normalization process.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:488]:
```suggestion
5. **Leverage documentation:** Leverage OpenSearch documentation and community resources for additional guidance on implementing and optimizing normalization techniques.
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:489]:
```suggestion
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:499]:
```suggestion
### References and further reading
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:503]:
```suggestion
1. [Hybrid search](https://opensearch.org/docs/latest/search-plugins/hybrid-search/)
```

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (natebower) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:504]:
```suggestion
2. [Normalization processor](https://opensearch.org/docs/latest/search-plugins/search-pipelines/normalization-processor/)
```

**Review by natebower — COMMENTED** [review]
@Lucas-Jeanniot @pajuric Editorial review complete. Please see my comments and changes and let me know if you have any questions. Apologies for the number of suggestions, but the vast majority of them are either capitalization or formatting changes. Thanks!

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (pajuric) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:9]: Please update the meta with the following: 
meta_keywords:  rank normalization, hybrid search, OpenSearch score normalization, search relevance, user experience

meta_description:  Learn about rank normalization in hybrid search and how to OpenSearch can help you standardize scores from diverse algorithms to deliver more accurate and fair search results.

File: _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md
Comment (pajuric) [inline _posts/2024-08-27-How-does-the-rank-normalization-work-in-hybrid-search.md:6]: Update to 2024-09-12

**Comment by pajuric** [comment]
@Lucas-Jeanniot - Please update the date to tomorrow, 19-Sept 2024 and we'll get this published.

**Comment by pajuric** [comment]
@nateynateynate @krisfreedain - Blog is ready to push live once the publish date is updated to 9/18/24

**Comment by kolchfa-aws** [comment]
Hi @Lucas-Jeanniot! Could you please resolve DCO by following [these instructions](https://github.com/opensearch-project/project-website/pull/3231/checks?check_run_id=30365085813) and we'll be ready to publish this blog. Thanks!

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @Lucas-Jeanniot! This was a smooth and very informative read. I fixed the formatting issues and we're ready to publish.

**Comment by navneet1v** [comment]
@Lucas-Jeanniot, very detailed blog. I like specially the section: `Comparing L2 and min-max normalization`

**Pull request merged**
